XCast/XCastCommon.h

```diff
@@ -24,8 +24,9 @@
 #define __XCAST_COMMON_H__
 /*
  * The maximum DIAL payload accepted per the DIAL 1.6.1 specification.
+ * payload from xdialserver is URI encoded
  */
-#define DIAL_MAX_PAYLOAD (4096)
+#define DIAL_MAX_PAYLOAD (4096*3)
```

Comment:
max payload allwayed is 4kb why multiplied by 3 which makes it 12kb?

Reply:
This payload here is base64 encoded, so it can require up to 3 times more memory then unencoded payload.